QUEST_LV_0100.tsv

```diff
@@ -6826,7 +6826,7 @@ QUEST_LV_0100_20150428_006825	Fortunately, it seems that you protected the altar
 QUEST_LV_0100_20150428_006826	The altar of Cheteque is destroyed, but you successfully activated the altar of Sviesa. Talk to Historian Cyrenia Odell.
 QUEST_LV_0100_20150428_006827	Obtained the fat of Miladu
 QUEST_LV_0100_20150428_006828	Rexipher kidnapped the historian, Cyrenia Odell. Go follow Rexipher to the Zachariel Intersection.
-QUEST_LV_0100_20150428_006829	As Rexipher ordered, after you defeated the protectors of the Mausoleum of Sesija parvis, Rexipher disappeared through the parvis. To find Cyrenia Odell, you should go past the parvis and go after Rexipher. He must be at the place not far away. 
+QUEST_LV_0100_20150428_006829	After you defeated the protector of the Mausoleum at Sesija Road as Rexipher ordered, Rexipher disappeared through the Road. To find Cyrenia Odell, you should go through the Road after Rexipher. He must be not far away. 
```

Comment:
protector -> guardian
Mausoleum -> Royal Mausoleum
Sesija Road? I think IMC misspelled the Korean. I think it's "Sesija Entrance" - I haven't done the quest yet, so I am not really sure. 
Road = parvise (https://en.wikipedia.org/wiki/Parvise)